vector/build.gradle

```diff
@@ -449,7 +449,11 @@ dependencies {
     // UnifiedPush
     implementation 'com.github.UnifiedPush:android-connector:1.2.0'
     // UnifiedPush gplay flavor only
-    gplayImplementation 'com.github.UnifiedPush:android-embedded_fcm_distributor:1.1.0'
+    gplayImplementation('com.github.UnifiedPush:android-embedded_fcm_distributor:1.1.0') {
+        exclude group: 'com.google.firebase', module: 'firebase-core'
+        exclude group: 'com.google.firebase', module: 'firebase-analytics'
+        exclude group: 'com.google.firebase', module: 'firebase-measurement-connector'
```

Comment:
It's weird that we have to exclude those modules again, since it's already done here: https://github.com/UnifiedPush/android-embedded_fcm_distributor/blob/main/lib/build.gradle#L44

Anyway, the CI is happy now, thanks

Reply:
Yes, that's very weird. And when the lib is a module of the project it isn't necessary. That's probably because of jitpack then